src/specs/01_basic/private_raw_smart_contract_spam.spec

```diff
@@ -0,0 +1,12 @@
+# Multiple private raw smart contracts between nodes signed externally
+
+ Tags: basic, spam, raw
```

Comment:
actually, we need to run this against Besu. with this, it won't work.

we need to change this line https://github.com/ConsenSys/quorum-acceptance-tests/blob/master/.github/workflows/run.yml#L125 to run this. you can change `!spam` to `(spam && raw)` and see if this tests runs as part of the Besu one

Reply:
as this is part of the EPIC for privacy interop with Besu

Reply:
updated